src/main.py

```diff
@@ -165,11 +190,12 @@ def reset(gen):
 					# Close the browser.

```

Comment:
The password reset page doesn't require any user input such as a Captcha so I found it useful to just close the browser and reopen